application-xpoll-test/application-xpoll-test-pageobjects/src/main/java/org/xwiki/xpoll/test/po/InPreparationStatusViewPage.java

```diff
@@ -25,15 +25,15 @@
 
 public class InPreparationStatusViewPage extends ViewPage
 {
-    @FindBy(xpath = "//div[@class = 'xform']//dd[1]")
+    @FindBy(xpath = "//div[@class = 'xform']/dl/dd")
     public WebElement pollDescription;
 
-    @FindBy(xpath = "//div[@class = 'xform']//dd[2]")
-    public WebElement pollStatus;
-
-    @FindBy(xpath = "//div[@class = 'xform']//dd[3]")
+    @FindBy(xpath = "//div[@class = 'xform']//dd[1]")
```

Comment:
Couple of mentions: 
- There is no test that tests for the values of the Proposals;
- This xpath targets multiple nodes. The luck is that Proposals it's the first elements it finds. If you were to target only Proposals this could have been a way `//div[@class = 'xform']//div[@class='row']//div[1]//dd`;

Reply:
I'm going to change the path right now.
Should I also add the test for Proposals?

Reply:
Yeap, otherwise there is no reason to have a path that is not used. Ty